src/index.js

```diff
@@ -54,7 +55,17 @@ export default function typescript ( options = {} ) {
 		throw new Error( `rollup-plugin-typescript: Couldn't process compiler options` );
 	}
 
-	const compilerOptions = parsed.options;
+	// let typescript load inheritance chain if there are base configs
+	const extendedConfig = !tsConfig.extends ?
+		null :
+		typescript.parseJsonConfigFileContent(tsConfig, typescript.sys, process.cwd(), parsed.options);
+	if (extendedConfig && extendedConfig.errors.length) {
+		extendedConfig.errors.forEach( error => console.error( `rollup-plugin-typescript: ${ error.messageText }` ) );
+
+		throw new Error( `rollup-plugin-typescript: Couldn't process compiler options` );
```

Comment:
and use [`this.error`](https://www.rollupjs.org/guide/en/#thiserrorerror-string--error-position-number---column-number-line-number---never) here, as it will let Rollup handle the output and abort the bundling process properly.